plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -578,17 +587,18 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
                 false,
             )
 
-            is CodeModernizerJobCompletedResult.JobFailedInitialBuild -> {
-                ApplicationManager.getApplication().invokeLater {
-                    runInEdt { BuildErrorDialog.create(result.failureReason) }
```

Comment:
How does this look? The content here is coming from the backend and may be verbose, thats why this one is a dialog instead of a notification as notifications truncates  text when generating the preview. Can you add some screenshots of this and is this dicussed with UX?

Reply:
+1 For before an after screenshots to help understand the full scope/context.

Reply:
Already discussed with UX (Allie) that we shouldn't be showing a dialog and a notification with the same content, which was happening so far.
![image](https://github.com/aws/aws-toolkit-jetbrains/assets/60020664/036d1212-372f-41a1-9cc8-0121ed88f464)

Reply:
The specific text is going to be reviewed & modified later. We previously had this notification + a modal in the middle showing the same exact info. Allie said we only need the notification. Following this logic, I removed the build error dialog / modal too and instead put the same info in a notification.

Reply:
<img width="975" alt="Screenshot 2024-03-13 at 10 46 02 AM" src="https://github.com/aws/aws-toolkit-jetbrains/assets/60020664/e8e923ad-1738-476b-b392-aeb6862f1bd3">

Reply:
before ^

Reply:
It's the exact same for the pre-build error.